data/json/npcs/TALK_CYBORG_1.json

```diff
@@ -2,10 +2,169 @@
   {
     "id": "TALK_CYBORG_1",
     "type": "talk_topic",
-    "dynamic_line": "You... You saved me!  *Zzzt* Let me come with you, I can help!",
+	"dynamic_line":
+	{
+        "npc_has_var": "cyborg_has_talked",
+		"type": "dialogue",
+		"context": "cyborg",
+		"value": "yes",
+		"no": "I... I'm free. *Zzzt* I'm actually free! *bzzz* Look, you're the first person I've seen in a long time.",
+		"yes": "Hey again. *kzzz*"
+	},
+  
```

Comment:
Consider adding:
```
    "speaker_effect": {
      "effect": { "npc_add_var": "cyborg_has_talked", "type": "dialogue", "context": "cyborg", "value": "yes" },
      "sentinel": "cyborg_has_talked"
    },
```
here, which will 100% of the time set the cyborg_has_talked var the first time you talk to the cyborg, and then remove all the npc_add_var effects afterwards.

Reply:
I was looking at doing it that way, but I wasn't 100% sure how it worked, so I played it safe.